deploy/docs/Best_Practices.md

```diff
@@ -124,6 +124,50 @@ excludepodRegex: "(dashboard.*|sumologic.*)"
 excludeNamespaceRegex: “(sumologic|kube-public)”
 ```
 
+To exclude specific log messages, you will need to edit the `logs.source.containers.conf` section in the `collection-sumologic` configmap. For example, let's say you wanted to exclude messages from your MongoDB pods that include the following:
+```
+.*connection accepted from.*
+.*authenticated as principal.*
+.*client metadata from.*
+```
+First, edit the configmap:
+```bash
+kubectl edit configmap collection-sumologic -n sumologic
+```
+Find the `logs.source.containers.conf` section and modify as follows:
+```yaml
+<filter containers.**>
+  @type record_transformer
+  enable_ruby
+  renew_record true
+  <record>
+    log    ${record["log"].split(/[\n\t]+/).map! {|item| JSON.parse(item)["log"]}.any? ? record["log"].split(/[\n\t]+/).map! {|item| JSON.parse(item)["log"]}.join("") : record["log"] rescue record["log"]}
+    stream ${[record["log"].split(/[\n\t]+/)[0]].map! {|item| JSON.parse(item)["stream"]}.any? ? [record["log"].split(/[\n\t]+/)[0]].map! {|item| JSON.parse(item)["stream"]}.join("") : record["stream"] rescue record["stream"]}
+    time   ${[record["log"].split(/[\n\t]+/)[0]].map! {|item| JSON.parse(item)["time"]}.any? ? [record["log"].split(/[\n\t]+/)[0]].map! {|item| JSON.parse(item)["time"]}.join("") : record["time"] rescue record["time"]}
+  </record>
+</filter>
+<filter containers.**>
+  @type grep
+  <regexp>
+    key hostname
+    pattern /^mongodb-\d+\./
+  </regexp>
+  <exclude>
+    key message
+    pattern /(.*connection accepted from.*|.*authenticated as principal.*|.*client metadata from.*)/
+  </exclude>
+</filter>
+```
+Apply the updated configmap:
+```bash
+kubectl apply configmap collection-sumologic -n sumologic
+```
+Bounce the fluentD pods (note that there will be multiple `collection-sumologic` pods that need to be restarted for the configmap update to take):
+```bash
+kubectl get pods -n sumologic
+kubectl delete pods <collection-sumologic-random_string>
```

Comment:
Alternatively you could use `kubectl rollout restart <deployment|statefulset>` to rollout the updates

Reply:
OK so how about we change it to:

```bash
kubectl get deployments -n sumologic
kubectl rollout restart <fluentd_deployment_nam3>
```